install.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
-#cp tools/pre-commit.pl .git/hooks/pre-commit
 [ -e .git/hooks/pre-commit ] ||  ln -s ../../tools/pre-commit.pl .git/hooks/pre-commit
-cpanm --installdeps --with-develop --with-recommends .
+cpanm App::cpm
+cpanm Carton::Snapshot
```

Comment:
Could probably already use `cpm` at this point, since you're not using the snapshot file here.

Reply:
note: this script is not used by the Dockerfile at this pont but for setting up a dev en on a freshly installed perl version